background.js

```diff
@@ -3,33 +3,33 @@ chrome.browserAction.setBadgeBackgroundColor({color: '#4688F1'});
 let reloads_holder = {};
 
 chrome.storage.onChanged.addListener((whatchanged, area) => {
-    if(area==="local"){
-        let keys = Object.keys(whatchanged);
-        keys.forEach(element => {
-            console.log(whatchanged[element]);
-            if(whatchanged[element].newValue){
-                reloads_holder[element] = setInterval(()=>{
-                    chrome.tabs.reload(parseInt(element));
-                }, 10000);
-            }
-            else {
-                clearInterval(reloads_holder[element]);
+    const currentKey = Object.keys(whatchanged)[0];
+    const currentVal = whatchanged[currentKey].newValue;
+    if(area === "local" && typeof currentVal === 'boolean' ) {
```

Comment:
Not a big thing, but to make it more intuitive to the reader, we first should only check if the change is in the local store and then care about what changed. Something like:

```suggestion
    if(area === "local" ) {
        const currentKey = Object.keys(whatchanged)[0];
        const currentVal = whatchanged[currentKey].newValue;
        if(typeof currentVal === 'boolean'){

```

Reply:
Done